src/components/VmActions/style.css

```diff
@@ -61,3 +61,10 @@
     height: 27px;
 }
 
+.menu-item-danger a {
+    color: #cc0000 !important;
```

Comment:
I prefer to avoid `!important`, here it is harmless, but it could soon become a habit. Also `!important` can be replaced with adding additional classes to declaration.

https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
Thanks! Good read!

Reply:
I still think it will be better to remove `!important`